proteobench/io/params/__init__.py

```diff
@@ -2,29 +2,30 @@
 from typing import Optional
 
 
+# Reference for parameter names
+# https://github.com/bigbio/proteomics-sample-metadata/blob/master/sdrf-proteomics/assets/param2sdrf.yml
+# new name as comment
 @dataclass
 class ProteoBenchParameters:
     search_engine: Optional[str] = None
     software_version: Optional[str] = None
-    fdr_psm: Optional[str] = None
-    fdr_peptide: Optional[str] = None
-    fdr_protein: Optional[str] = None
-    MBR: Optional[str] = None
-    precursor_tol: Optional[str] = None
+    fdr_psm: Optional[str] = None  # ident_fdr_psm
+    fdr_peptide: Optional[str] = None  # ident_fdr_peptide
+    fdr_protein: Optional[str] = None  # ident_fdr_protein
+    MBR: Optional[str] = None  # enable_match_between_runs
+    precursor_tol: Optional[
+        str
+    ] = None  # precursor_mass_tolerance, value and unit not separated
     precursor_tol_unit: Optional[str] = None
-    fragment_tol: Optional[str] = None
+    fragment_tol: Optional[
+        str
+    ] = None  # fragment_mass_tolerance, value and unit not separated
     fragment_tol_unit: Optional[str] = None
-    enzyme_name: Optional[str] = None
-    missed_cleavages: Optional[str] = None
-    min_pep_length: Optional[str] = None
-    max_pep_length: Optional[str] = None
-    fixed_modifications: Optional[str] = None
-    variable_modifications: Optional[str] = None
-    max_num_modifications: Optional[str] = None
-    precursor_charge: Optional[str] = None
-
-
-# params = ProteoBenchParameters()
-# params.search_engine = "MaxQuant"
-# params
-# %%
+    enzyme_name: Optional[str] = None  # enzyme
+    missed_cleavages: Optional[str] = None  # allowed_miscleavages
+    min_pep_length: Optional[str] = None  # min_peptide_length
+    max_pep_length: Optional[str] = None  # max_peptide_length
+    fixed_modifications: Optional[str] = None  # fixex_mods
+    variable_modifications: Optional[str] = None  # variable_mods
+    max_num_modifications: Optional[str] = None  # max_mods
+    precursor_charge: Optional[str] = None  # min_precursor_charge
```

Comment:
if we don't enter all the possible precursor charges, then we should have min_ and max_precursor_charge

Reply:
I am not sure. I guess that if this happens, we'll have a missing value there.
What I meant was more that in cases like in MQ, where they give all the charges that they consider, we need to get the min. and max. No?

Reply:
MQ specifies a range of min. and max. charge. We only have a problem if a tools would e.g. allow peptides with charges 2,3 and 5, but excludes charges with 4. But I guess this is quite uncommon?

Reply:
In that case yes, as we should not diverge from SDRF-extended.